inc/compatibility/class-fastvelocityminify.php

```diff
@@ -41,5 +33,6 @@ public function apply_fix() {
 	/**
 	 * @return void
 	 */
-	public function remove_fix() {}
+	public function remove_fix() {
+	}
```

Comment:
Did linting flag this? I would expect `{}` to still pass.

```suggestion
	public function remove_fix() {}
```